### PR TITLE
GRO-217: Auctions Landing Page QA Adjustments

### PR DESCRIPTION
**Pull request by The-Beez-Kneez**
These are QA adjustments for the auctions2 landing page tabs. 

Following update to Metaphysics to filter auctions based on `auctionState: CLOSED, OPEN, UPCOMING`. Readjusted graphql requests based on `auctionStatus`.

New Upcoming Auction Display:
Screen Shot 2021-03-23 at 2 54 28 PM

New Past Auction Display:
Screen Shot 2021-03-23 at 2 54 37 PM

New Current Auction Display:
Screen Shot 2021-03-23 at 2 54 18 PM

### CODE REVIEW

File: src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx [hunk 1]
```diff
@@ -49,14 +49,19 @@ export const AuctionArtworksRail: React.FC<AuctionArtworksRailProps> = ({
...
-
-        <Box height={AUCTION_ARTWORKS_RAIL_HEIGHT}>
-          {isEnteredView ? (
-            <AuctionArtworksRailArtworksQueryRenderer id={sale.internalID} />
-          ) : (
-            <AuctionArtworksRailPlaceholder />
-          )}
-        </Box>
+        {/* Only showing Artwork Images for Current and Past Auctions */}
+        {(!sale.isLiveOpen && sale.status == "open") ||
```
Comment (damassi) [inline src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx:53]: In JS we'll always want to be sure to use [strict equality](https://stackoverflow.com/questions/523643/difference-between-and-in-javascript)

File: src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx [hunk 2]
```diff
@@ -49,14 +49,19 @@ export const AuctionArtworksRail: React.FC<AuctionArtworksRailProps> = ({
...
+          <Box height={AUCTION_ARTWORKS_RAIL_HEIGHT}>
+            {isEnteredView ? (
+              <AuctionArtworksRailArtworksQueryRenderer id={sale.internalID} />
+            ) : (
+              <AuctionArtworksRailPlaceholder />
+            )}
+          </Box>
+        ) : (
+          <></>
+        )}
```
Comment (damassi) [inline src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx:64]: We could simplify this logic a bit by collapsing some of the args and removing the ternary:
```tsx
{!sale.isLiveOpen && (sale.status === "open" || sale.status === "closed) && (
  <Box>...</Box>
)}
```

Comment (damassi) [reply src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx:64]: Or better yet, move the logic into a self-documenting variable name:

```
const isCurrentOrPastAuction = !sale.isLiveOpen && (sale.status === "open" || sale.status === "closed)

...

{isCurrentOrPastAuction && (
  <Box>...</Box>
)}

**Review by damassi — APPROVED** [review]
A couple minor requests but once those are in LGTM 👍 

(After this is merged and we do some QA on staging we'll want to get some tests in as follow-up.)

File: src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx (hunk 1 above)
Comment (The-Beez-Kneez) [reply src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx]: Removed this.

File: src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx (hunk 2 above)
Comment (The-Beez-Kneez) [reply src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx]: Removed this also.

File: src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx [hunk 3]
```diff
@@ -72,6 +71,9 @@ export const AuctionArtworksRailFragmentContainer = createFragmentContainer(
         href
         name
         formattedStartDateTime
+        isLiveOpen
+        status
+        isPreliminary
```
Comment (damassi) [inline src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx:76]: Are these fields being used anywhere in the component?

File: src/v2/Apps/Auctions/Routes/UpcomingAuctions.tsx [hunk 4]
```diff
@@ -99,6 +98,7 @@ export const UpcomingAuctionsPaginationContainer = createPaginationContainer(
               status
               formattedStartDateTime
               eventStartAt
+              isLiveOpen
```
Comment (damassi) [inline src/v2/Apps/Auctions/Routes/UpcomingAuctions.tsx:101]: Is this being used?

**Comment by damassi** [comment]
@The-Beez-Kneez - can we add a screenshot or screen recording of the text-only titles within the upcoming tab?

File: src/v2/Apps/Auctions/Routes/UpcomingAuctions.tsx (hunk 4 above)
Comment (The-Beez-Kneez) [reply src/v2/Apps/Auctions/Routes/UpcomingAuctions.tsx:101]: Removed

File: src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx (hunk 3 above)
Comment (The-Beez-Kneez) [reply src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx]: Removed.

**Comment by The-Beez-Kneez** [comment]
@damassi I actually have a question about the Upcoming auctions without images that I need to ask Nicole and Jenna. Currently do not have it implemented, as the old logic was no longer suitable with the new filtering change.

**Comment by The-Beez-Kneez** [comment]
@damassi after discussion with Nicole and Jenna, we are going to display artwork images for Upcoming Auctions, in the same manner as Past Auctions and Current Auctions.

**Review by jonallured — APPROVED** [review]
LGTM!! 🎉

**Pull request merged**